### PR TITLE
Add threshold and sensitivity control

### PR DESCRIPTION
**Pull request by caternuson**
Adds new features for #6. Threshold settings can be done per channel or globally for all channels. Also adds `sensitivity`, which is a global only setting.

```python
Adafruit CircuitPython 3.0.3 on 2018-10-10; Adafruit ItsyBitsy M4 Express with samd51g19
>>> import board, busio
>>> from adafruit_cap1188.i2c import CAP1188_I2C
>>> i2c = busio.I2C(board.SCL, board.SDA)
>>> cap = CAP1188_I2C(i2c)
>>> cap.thresholds
(64, 64, 64, 64, 64, 64, 64, 64)
>>> cap.thresholds = 99
>>> cap.thresholds
(99, 99, 99, 99, 99, 99, 99, 99)
>>> cap[3].threshold
99
>>> cap[3].threshold = 123
>>> cap.thresholds
(99, 99, 123, 99, 99, 99, 99, 99)
>>> cap.sensitivity
32
>>> cap.sensitivity = 128
>>> cap.sensitivity
128
>>> 
```

### CODE REVIEW

File: adafruit_cap1188/cap1188.py [hunk 1]
```diff
@@ -63,16 +63,21 @@
                               const(0x15),
                               const(0x16),
                               const(0x17))
+CAP1188_SENSITIVTY          = const(0x1F)
```
Comment (tannewt) [inline adafruit_cap1188/cap1188.py]: Shouldn't these all start with `_` to save memory? I realize you didn't change it here but its probably worth doing. Variable names take a lot of memory.

Comment (caternuson) [reply adafruit_cap1188/cap1188.py]: Yep. Just not being good at being consistent. Names changed.

**Review by ladyada — APPROVED** [review]
good find!

**Review by kattni — APPROVED** [review]
Looks good, thanks for adding this and fixing it up!

**Pull request merged**